### PR TITLE
Info about conductivity now mentions item, not weapon

### PR DESCRIPTION
**Pull request by Night-Pryanik**
As I found it strange to see "This _weapon_ doesn't conduct electricity" about a book or a piece of clothing.

### CODE REVIEW

**Comment by pisskop** [comment]
the only context in which conductivity matters is when used as a weapon

File: src/item.cpp [hunk 1]
```diff
@@ -1556,6 +1546,16 @@ std::string item::info( bool showtext, std::vector<iteminfo> &info ) const
...
+        if( !conductive () ) {
+                info.push_back( iteminfo( "BASE", string_format( _( "* This item <good>does not conduct</good> electricity." ) ) ) );
+            }
+            else if( has_flag( "CONDUCTIVE" ) ) {
+                info.push_back( iteminfo( "BASE", string_format( _( "* This item effectively <bad>conducts</bad> electricity, as it has no guard." ) ) ) );
+            }
+            else {
+                info.push_back( iteminfo( "BASE", string_format( _( "* This item <bad>conducts</bad> electricity." ) ) ) );
+            }
+
```
Comment (Toothspit) [inline src/item.cpp]: Everything has an extra indent, the closing bracket should be directly below the if, right?

**Comment by kevingranade** [comment]
Please create the branch on your own repository, the fact that the permissions allow you to create it in the main repository is a shortcoming of the github permissions.

**Comment by Night-Pryanik** [comment]
Should I open new PR (and a new ones for #21125 and #21011), or it is a remark for the future?

**Comment by kevingranade** [comment]
Just plan to do it in the future, it's only really an issue if a lot of people started doing it, so I want to avoid setting a precedent.

**Comment by Night-Pryanik** [comment]
OK, noted.

**Pull request merged**